src/main/java/seedu/address/model/event/Consultation.java

```diff
@@ -0,0 +1,133 @@
+package seedu.address.model.event;
+
+import java.io.File;
+import java.time.LocalDate;
+import java.util.List;
+
+import seedu.address.model.person.Person;
+
+/**
+ * Allows the TA to create a Consultation event that usually occurs once a week.
+ * Currently, there are no different behaviours between Consultation, Consultations and Consultation.
+ * Custom behaviours will be added in future versions / milestones.
+ */
+public class Consultation extends Event {
+
+    public Consultation(String name) {
+        super(name);
+    }
+
+    public Consultation(String name, List<Person> students) {
+        super(name, students);
+    }
+
+    public Consultation(String name, LocalDate eventDate, List<Person> students) {
+        super(name, eventDate, students);
+    }
+
+    public Consultation(String name, LocalDate eventDate, List<Person> students, List<File> attachments) {
+        super(name, eventDate, students, attachments);
+    }
+
+    public Consultation(String name, LocalDate eventDate, List<Person> students, List<File> attachments,
+                    List<Note> notes) {
+        super(name, eventDate, students, attachments, notes);
+    }
+
+    public String getName() {
+        return super.getName();
+    }
+
+    /**
+     * Gets the list of students
+     * @return students
+     */
+    public List<Person> getStudents() {
+        return super.getStudents();
+    }
+
+    /**
+     * Adds a specific student into the list of students
+     * @param student
+     */
+    public void addStudent(Person student) {
+        super.addStudent(student);
+    }
+
+    /**
+     * Removes a specific student from the list of students
+     * @param student
+     */
```

Comment:
An empty line needs to be included between the description and parameter section according to our style guide!

Reply:
Ok, I have resolved it